lib/rules/attributes-order.js

```diff
@@ -200,17 +206,27 @@ function create(context) {
     [ATTRS.UNIQUE, ATTRS.SLOT],
     ATTRS.TWO_WAY_BINDING,
     ATTRS.OTHER_DIRECTIVES,
-    ATTRS.OTHER_ATTR,
+    [ATTRS.OTHER_ATTR_DYNAMIC, ATTRS.OTHER_ATTR_STATIC],
     ATTRS.EVENTS,
     ATTRS.CONTENT
   ]
   if (context.options[0] && context.options[0].order) {
-    attributeOrder = context.options[0].order
+    attributeOrder = [...context.options[0].order]
   }
   const alphabetical = Boolean(
     context.options[0] && context.options[0].alphabetical
   )
 
+  for (const [index, item] of attributeOrder.entries()) {
+    if (Array.isArray(item) && item.includes(ATTRS.OTHER_ATTR)) {
+      const attributes = item.filter((i) => i !== ATTRS.OTHER_ATTR)
+      attributes.push(...otherAttrs)
+      attributeOrder[index] = attributes
+    } else if (item === ATTRS.OTHER_ATTR) {
+      attributeOrder[index] = otherAttrs
+    }
```

Comment:
Maybe that's more readable?

```suggestion
    if (item === ATTRS.OTHER_ATTR) {
      attributeOrder[index] = otherAttrs
    } else if (Array.isArray(item)) {
      const itemIndex = item.indexOf(ATTRS.OTHER_ATTR)
      if (itemIndex > -1) {
        attributeOrder[index].splice(itemIndex, 1, ...otherAttrs)
      }
    }
```

Reply:
I tried to do it this way, but it doesn't work.
Whenever I try to change some item of attributeOrder, I get an error